Translation[CN]: Basics, Numbers, Strings

glenntfung opened the pull request:
My plan is to wait until Basics, Numbers, Strings are translated to mark this PR as completed.

sumn2u commented:
@glenntfung Thanks for this draft PR. Quick question, are you also going to work on the ` Basics`, `Numbers` and `Strings`?

glenntfung commented:
These three parts are finished. I rebased so the commit date of the first commit changed :(

I also fixed a typo that could cause confusion in a separate commit in the English version I spotted. 

**Known issues**: links to untranslated files.
